src/uct/tcp/tcp_cm.c

```diff
@@ -86,6 +86,17 @@ uct_tcp_cm_conn_pkt_check_event(const uct_tcp_ep_t *ep,
     return UCS_OK;
 }
 
+static void uct_tcp_cm_trace_somaxconn_exceed()
```

Comment:
I think it makes sense to rename to "uct_tcp_cm_log_somaxconn_exceed" pass log level to this function and call it with error

Reply:
thanks! nice point - done